src/RPL-1.5.xml

```diff
@@ -200,7 +200,7 @@
         <item>
             <bullet>4.0</bullet>
           Grant of License From Contributor. By application of the provisions in Section 6 below, each
-             Contributor hereby grants You a world-wide, royalty-<alt match=" ?" name="extraSpace"/>free, non-exclusive license, subject to
+             Contributor hereby grants You a world-wide, <alt match="royalty ?- ?free" name="extraSpace">royalty-free</alt>, non-exclusive license, subject to
```

Comment:
I like these better with the shorter `<alt>`, since I expect the marked-up HTML will be easier to read with a single character of variable text.

And even with the longer `<alt>`, I see no reason to allow a space before the hyphen.

Reply:
> I like these better with the shorter <alt>, since I expect the marked-up HTML will be easier to read with a single character of variable text.

Me to, but the tools will not currently process this correctly.

> I see no reason to allow a space before the hyphen.

Removed